assets/css/main.css

```
@@ -604,14 +621,33 @@ button {
+   width — and vertically filling the row, which is one dot tall. Placed with
+   left and a margin rather than a transform, because the bounce below needs the
+   transform to itself. */
 .hero-greeting__dot {
-    display: inline-block;
+    position: absolute;
+    top: 0;
+    width: var(--dot-size);
+    height: var(--dot-size);
+    margin-left: calc(var(--dot-size) / -2);
+    border-radius: 50%;
+    background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)